src/legacy/core_plugins/kibana/index.js

```diff
@@ -34,6 +34,8 @@ import { getUiSettingDefaults } from './ui_setting_defaults';
 import { registerCspCollector } from './server/lib/csp_usage_collector';
 import { injectVars } from './inject_vars';
 import { i18n } from '@kbn/i18n';
+// eslint-disable-next-line
+import { AppCategory } from '../../../core/public/application/types';
```

Comment:
as `AppCategory` is used both from client and server code, it should probably be put in `src/core/types` instead of `src/core/public` to avoid these eslint warnings (we are working on a more functional/module code separation than public/server, but for now this is probably the best compromise)